lib/internal/Magento/Framework/Locale/TranslatedLists.php

```diff
@@ -150,10 +157,20 @@ public function getOptionCurrencies()
         $currencies = (new CurrencyBundle())->get($this->localeResolver->getLocale())['Currencies'] ?: [];
         $options = [];
         $allowed = $this->_config->getAllowedCurrencies();
+        $selectedCurrencies = explode(
+            ',', $this->scopeConfig->getValue(
+                'system/currency/installed',
```

Comment:
I would put inside private const inside class

Reply:
Not needed until it is used only once.